Build on the ARM machine

**Describe the bug**
The container image can not be created on arm machine (such as raspberry pi).

**To Reproduce**

0. Prepare a raspberry pi board.
1. Run `make distclean`
2. Run `make create_context CONFIGFILE=armc`
3. Run `make`

**Log**
```
pi@raspberrypi:~/edge-home-orchestration-go $ make distclean

--------------------------------------
  Build clean
--------------------------------------
make[1]: Entering directory '/home/pi/edge-home-orchestration-go/examples/native'
make[1]: Leaving directory '/home/pi/edge-home-orchestration-go/examples/native'
```
```
pi@raspberrypi:~/edge-home-orchestration-go $ make create_context CONFIGFILE=armc
```
```
pi@raspberrypi:~/edge-home-orchestration-go $ make
make clean
make[1]: Entering directory '/home/pi/edge-home-orchestration-go'

--------------------------------------
  Build clean
--------------------------------------
make[2]: Entering directory '/home/pi/edge-home-orchestration-go/examples/native'
make[2]: Leaving directory '/home/pi/edge-home-orchestration-go/examples/native'
make[1]: Leaving directory '/home/pi/edge-home-orchestration-go'

--------------------------------------
  Go Mod Vendor
--------------------------------------
GO111MODULE=on go mod vendor

--------------------------------------
  Create Executable binary
--------------------------------------
GOARM=7 GOARCH=arm GO111MODULE=on go build -a -ldflags '-extldflags "-static" -X main.version= -X main.commitID=dcf250e -X main.buildTime=20210716.1501 -X main.buildTags=' -o /home/pi/edge-home-orchestration-go/bin/edge-orchestration /home/pi/edge-home-orchestration-go/cmd/edge-orchestration/main.go || exit 1
total 18336
drwxr-xr-x  3 pi pi     4096 Jul 16 15:05 .
drwxr-xr-x 18 pi pi     4096 Jul 16 15:01 ..
-rwxr-xr-x  1 pi pi 18762294 Jul 16 15:05 edge-orchestration
drwxr-xr-x  2 pi pi     4096 Jul  5 11:09 qemu
make build_docker_container
make[1]: Entering directory '/home/pi/edge-home-orchestration-go'

--------------------------------------
  Create Docker container arm32v7
--------------------------------------
docker rm -f "edge-orchestration"
Error: No such container: edge-orchestration
docker rmi -f "edge-orchestration":"coconut"
Error: No such image: edge-orchestration:coconut
cp: cannot stat '/usr/bin/qemu-arm-static': No such file or directory
make[1]: *** [Makefile:165: build_docker_container] Error 1
make[1]: Leaving directory '/home/pi/edge-home-orchestration-go'
make: *** [Makefile:241: all] Error 2

```

**Test environment configuration (please complete the following information):**
* Firmware version: Raspberry Pi OS
* Hardware: ARM
* Edge Orchestration Release: Coconut

--- a/Makefile
+++ b/Makefile
@@ -163,10 +163,14 @@ build_docker_container:
 	-docker rmi -f $(DOCKER_IMAGE):$(CONTAINER_VERSION)
 	$(Q) mkdir -p $(BASE_DIR)/bin/qemu
 ifeq ($(CONFIG_ARM),y)
+ifneq ($(shell uname -m),armv7l)
 	$(Q) cp /usr/bin/qemu-arm-static $(BASE_DIR)/bin/qemu
 endif
+endif
 ifeq ($(CONFIG_ARM64),y)
+ifneq ($(shell uname -m),aarch64)
 	$(Q) cp /usr/bin/qemu-aarch64-static $(BASE_DIR)/bin/qemu
+endif
 endif
 	$(DOCKER) build --tag $(PKG_NAME):$(CONTAINER_VERSION) --file $(BASE_DIR)/Dockerfile --build-arg PLATFORM=$(CONTAINER_ARCH) .
 	-docker save -o $(BASE_DIR)/bin/edge-orchestration.tar edge-orchestration

